apps/chat/channels.py

```diff
@@ -489,7 +489,7 @@ def _create_new_experiment_session(self):
         )
 
     def _is_reset_conversation_request(self):
-        return self.user_query == ExperimentChannel.RESET_COMMAND
+        return self.user_query.lower() == ExperimentChannel.RESET_COMMAND
```

Comment:
How about also adding `.strip()` to remove any whitespace?

Reply:
https://github.com/dimagi/open-chat-studio/pull/877/commits/0cb7d48c827e6a88961d6da2932f103b7b3a1bee